tests/foreman/api/test_activationkey.py

```diff
@@ -173,6 +212,24 @@ def test_negative_update_1(self):
                 actual = {attr: getattr(act_key, attr) for attr in want.keys()}
                 self.assertEqual(want, actual)
 
+    def test_negative_update_2(self):
+        """@Test: Create activation key then update its name to an invalid name.
+
+        @Assert: Activation key is created, and its name is not updated.
+
+        @Feature: ActivationKey
+
```

Comment:
No need to add this blank line.

For more info check: https://www.python.org/dev/peps/pep-0008/#documentation-strings

PS: I know that we have been doing this for a long time, but some time ago I read again that section and figured out that we ensured a non PEP8 standard.